carfield.mk

```diff
@@ -102,7 +102,7 @@ endif
 ######################
 
 CAR_NONFREE_REMOTE ?= git@iis-git.ee.ethz.ch:carfield/carfield-nonfree.git
-CAR_NONFREE_COMMIT ?= 6a962ae5f6eca3430103da57c7597afa503a1c39
+CAR_NONFREE_COMMIT ?= e45450d8ae31127dc2b00ba70e1569c9f495387b
```

Comment:
can you please rebase your branch in carfield-nonfree on top of main and update this pointer? :)

Reply:
Done